src/qt/guiutil.cpp

```diff
@@ -58,9 +58,10 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 
-#include <objc/objc-runtime.h>
 #include <CoreServices/CoreServices.h>
 #include <QProcess>
+
+void ForceActivation();
```

Comment:
What are benefits of doing that? A header including will increase the current translation unit, however.

Reply:
When I noticed the function declaration it was not immediately obvious why the corresponding definition was not in the same cpp file. It was only obvious why it is as it is after I found the corresponding function definition in the `mackdockiconhandler.mm` file. 

I thought that by declaring the function in the `mackdockiconhandler.h` header file and keeping the declaration and definition close to each other, it would save the next person from wondering the same.

But perhaps it just me. I am afraid I don't have a strong argument.